week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,18 +1,37 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+<<<<<<< Updated upstream
```

Comment:
It looks like something weird happened with this file - can you fix it up? :)

The easiest way may be to undo your changes by running:
```
git checkout master -- 2-logic-error.js
git reset HEAD -- 2-logic-error.js
```

and then re-doing the exercise